### PR TITLE
chore(Dockerfile): RHICOMPL-1369 use the minimal ubi8 image

### PR DESCRIPTION
**Pull request by skateman**
Old size: 1.12GB
New size: 595MB

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,55 @@
...
 
-ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true QMAKE=/usr/lib64/qt5/bin/qmake DISABLE_ASSET_COMPILATION=true
+COPY --chown=1001:0 . /opt/app-root/src
+COPY --chown=1001:0 --from=build /opt/app-root/src/.bundle /opt/app-root/src/.bundle
+
+RUN ./scripts/fix-permissions ./
 
-# Install the dependencies
-RUN /usr/libexec/s2i/assemble
+ENV RAILS_ENV=production RAILS_LOG_TO_STDOUT=true HOME=/opt/app-root/src devDeps=$devDeps
```
Comment (vkrizan) [inline Dockerfile:53]: Nit: upercase for env var `devDeps` -> `DEV_DEPS`

**Review by vkrizan — APPROVED** [review]
LGTM. Nice!

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
369fee89-4831-47b5-8ce0-c1d84ad0c106
d6bb1d83-c01f-481f-9b47-efafd2d204dd

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
29f22d53-460d-4775-90f8-714c5fee03a4
c4584b71-e054-4c24-8193-2e6b725ac508

**Pull request merged**